docs/src/main/sphinx/release/release-366.md

```diff
@@ -0,0 +1,43 @@
+# Release 366 (XX Dec 2021)
+
+## General
+         
+* Add support for query retries. This feature can be turned on by setting the `retry-policy` 
+  config property or the `retry_policy` session property to `retry`. ({issue}`9361`)
+* Reduce queries lock contention. ({issue}`10246`, {issue}`10239`)
+* Query performance improvements. ({issue}`10225`)
```

Comment:
do we need to include this in RL? may be add some more context? If it's hard to condense in a couple of sentences, how about 

```
* Improve query performance through optimizations for in-memory representation. ({issue}`10225`)
```